### PR TITLE
Update FIPS140-3 exclusions with Windows

### PR DESCRIPTION
**Pull request by JinhangZhang**
Add failures shows up in the windows platform.

See https://github.com/ibmruntimes/openj9-openjdk-jdk23/pull/28

### CODE REVIEW

**Comment by JinhangZhang** [comment]
@jasonkatonica @llxia FYI

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt [hunk 1]
```diff
@@ -20,962 +20,972 @@
...
+com/sun/crypto/provider/TLS/TestPRF12.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/crypto/provider/TLS/TestPremaster.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/ShortECDSA.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/SignatureKeyInfo.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+com/sun/org/apache/xml/internal/security/TruncateHMAC.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/foreign/TestFallbackLookup.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/foreign/TestMismatch.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
```
Comment (keithc-ca) [inline test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:162]: These don't seem related to FIPS140-3 and they're not listed in other exclude lists: why are they here?
Same question for:
* java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java
* serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java
* serviceability/jvmti/vthread/GetThreadState/GetThreadStateTest.java
* serviceability/jvmti/vthread/SuspendResume2/SuspendResume2.java

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt [hunk 2]
```diff
@@ -20,962 +20,972 @@
...
+jdk/security/jarsigner/Properties.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+jdk/security/jarsigner/Spec.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+jdk/security/logging/RecursiveEventHelper.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+jdk/security/logging/TestSecurityPropertyModificationLog.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+jdk/security/logging/TestTLSHandshakeLog.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+serviceability/jvmti/GetOwnedMonitorInfo/GetOwnedMonitorInfoTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,windows-all
+serviceability/jvmti/stress
```
Comment (keithc-ca) [inline test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:991]: Please add these keeping the list of test names for each group remain sorted.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 1 above)
Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:162]: 1. Failures are not identical among various JDK versions. 2. Those failures shows up while running in FIPS 140-3 mode. We had a discussion with the test team, test samples in some folders do not need to be run, so failed samples in these folders can be eliminated.

Comment (keithc-ca) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:162]: I was referring to other exclude lists for this repository (same version).

Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:162]: Im not quite sure what do you mean by other exclude lists for this repository. There is only one exclude list file for fips 140-3 right now.

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 2 above)
Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:991]: Updated and sorted the whole list.

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 1 above)
Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:162]: Removed those unrelated failures.

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt [hunk 3]
```diff
@@ -20,962 +20,964 @@
...
+java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/lang/SecurityManager/CheckSecurityProvider.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/lang/String/CompactString/NegativeSize.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,windows-all
+java/lang/reflect/records/IsRecordTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/lang/reflect/records/RecordPermissionsTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/lang/reflect/records/RecordReflectionTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
+java/lang/runtime/ObjectMethodsTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk23/issues/20 aix-all,linux-ppc64le,linux-s390x,linux-x64,windows-all
```
Comment (keithc-ca) [inline test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:169]: There are still several tests, like these, excluded here, but not in `ProblemList.txt` nor in `ProblemList-FIPS140_2.txt`. They really ought not to be excluded here, but they will be tested elsewhere so this isn't serious - I trust they be un-excluded in time.

**Review by keithc-ca — COMMENTED** [review]
Please squash.

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 3 above)
Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt]: updated

**Comment by JinhangZhang** [comment]
> Please squash.

Squashed. @llxia FYI

File: test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt (hunk 3 above)
Comment (keithc-ca) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt]: I wasn't asking you to remove these; they were only meant to be examples (there are many others that don't appear to be security related). It's fine that you removed them, but, sorry, I think `CheckSecurityProvider.java` should stay (it's also excluded in `ProblemList-FIPS140_2.txt`) unless you know that it passes.

Comment (JinhangZhang) [reply test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt:169]: Update the list and add them back.

**Comment by keithc-ca** [comment]
Jenkins line endings check

**Review by keithc-ca — APPROVED** [review]
